service-app/internal/api/client.go

```diff
@@ -1,75 +1,98 @@
 package api
 
 import (
-	"bytes"
 	"encoding/json"
 	"fmt"
-	"net/http"
 	"time"
 
+	"github.com/ministryofjustice/opg-scanning/config"
+	"github.com/ministryofjustice/opg-scanning/internal/httpclient"
+	"github.com/ministryofjustice/opg-scanning/internal/logger"
 	"github.com/ministryofjustice/opg-scanning/internal/types"
 )
 
-func CreateStubCase(url string, set types.BaseSet) (*types.ScannedCaseResponse, error) {
-	var scannedCaseRequest types.ScannedCaseRequest
+type CreateStubCase struct {
+	config     config.Config
+	logger     logger.Logger
+	httpClient *httpclient.HttpClient
+}
```

Comment:
Would this struct be better named as `Sirius`, since that's what it's creating an interface for?

Reply:
Actually its been change in the other PR to **Client**, as thats the generic handler.